.github/workflows/test.yml

```
@@ -0,0 +1,170 @@
+name: Node CI
+
+on:
+  push:
+    branches:
+      - "*"
+      - "*/*"
+
+jobs:
+  prepare:
+    name: Install Dependencies
+    runs-on: ubuntu-18.04
+    if: "!(contains(github.actor, 'process-engine-ci') || contains(github.actor, 'admin'))"
+
+    steps:
+    - uses: actions/checkout@master
+
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+
+    - name: Install Dependencies
+      working-directory: ./_integration_tests
+      run: |
+        npm i
```

Die package-lock ist hier eigentlich insgesamt eher überflüssig.
In den dev setups wurde die noch nie verwendet und ich wüsste grade auch nicht wozu.
Reicht eigentlich völlig aus, wenn wir die in der Runtime archivieren.

```suggestion
        npm i --no-package-lock
```